src/index.html

```diff
@@ -6,8 +6,41 @@
     <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Hello Mate Academy</h1>
+
+    <ul class="tree">
+      <li>Fruit</li>
+      <ul>
+        <li>Red</li>
+        <ul>
+          <li>Cherry</li>
+          <li>Strawberry</li>
+          <li>Watermelon</li>
+          <li>Pomegranate</li>
+        </ul>
```

Comment:
why didn't you wrap `ul` with `li`???
It is not valid HTML

Reply:
fixed